services/s3_service.py

```diff
@@ -88,14 +80,24 @@ def _get_users_from_org_people_file(self) -> list:
         return users
 
     def _download_file(self, object_name: str, file_path: str):
-        self.client.download_file(
-            self.bucket_name, object_name, file_path)
+        self.client.download_file(self.bucket_name, object_name, file_path)
         if not os.path.isfile(file_path):
             raise ValueError(
-                f"The {file_path} file did not download or is not in the expected location")
+                f"The {file_path} file did not download or is not in the expected location"
+                )
 
     def _upload_file(self, object_name: str, file_path: str):
         self.client.upload_file(file_path, self.bucket_name, object_name)
 
     def _delete_file(self, object_name: str):
         self.client.delete_object(Bucket=self.bucket_name, Key=object_name)
+
+    def is_well_known_mta_sts_enforce(self, domain: str) -> bool:
```

Comment:
You should add a unit test for this function to the test_s3_service.py test suite. 2 cases should be accounted for, where mta-sts is enabled and where it is disabled.